functions.php

```diff
@@ -936,7 +936,7 @@ function async_post($url, $payload)  {
 
 function sms_chunk_split($msg) {
     $msg = preg_replace('/[\r\n]+/', ' ', $msg);
-    $chunks = wordwrap($msg, 1600, '\n');
+    $chunks = wordwrap($msg, 149, '\n');
```

Comment:
why 149?

Reply:
sorry it actually could probably be 150, wanted to limit texts to 160characters so it would work on all handsets, alternatively i could make it 1589 or 1590. im splitting the texts up into chunks and numbering them so text could be '(1 of 10)' which would take up 9 characters

Reply:
I was under the impression that Twilio handled all of this and it was unnecessary to break things up.

Reply:
they do but only up 10 1600 characters after that error is thrown, so we could either split message by 1600 characters or so or split up by 160 or so. up to you

Reply:
they'll only auto concatenate up to 1600 characters

Reply:
in the case of this, the chunks make the user experience not ideal.  most people are going to get the entire message, it seems like Twilio and the carrier know how best to handle the message and we should leverage that.  Versus us baking in logic into Yap that will never be better than Twilio's.

Reply:
ok so we split into roughly 1600 character chunks then? alot of the time it would never be split. however you cant have more than 1600 characters in a body message. and today it is.

Reply:
split it on `1575`

Reply:
cool

Reply:
can you make `1575` a constant?

Reply:
this line wasnt needed `$msg = preg_replace('/[\r\n]+/', ' ', $msg);` i checked all languages and they look fine with out it